src/s1reader/utils/plot_bursts.py

```diff
@@ -0,0 +1,140 @@
+import argparse
+import glob
+from s1reader.s1_orbit import get_orbit_file_from_list
+from s1reader.s1_reader import load_bursts
+from osgeo import gdal, osr
+from shapely.geometry import Polygon
+import pandas as pd
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import folium
+from shapely import wkt
+import fiona
```

Comment:
pandas, geopandas, folium, and fiona are currently not part module requirements. I'm not sure how to proceed. AFAICT there's no need to require them for the production environment. @hfattahi what do you think?

Wrap `try-except`?

Do something like [ISCE3](https://github.com/isce-framework/isce3/tree/develop/.ci/images/centos), where we have different requirement files for documentation sake?

Reply:
@LiangJYu For this PR, try-except is fine with me. Yes this is just extra and many users including isce3 modules and compass don't need it. So the requirement should be kept light and we should document that for plotting purpose those extra packages should be installed or have another requirement file. Perhaps you could do those clarifications after this PR is merged.